contessa/migration.py

```diff
@@ -1,8 +1,7 @@
-# Python script that will apply the migrations up to head
 from contessa.db import Connector
 from packaging.version import parse as pv
 
-ALEMBIC_TABLE = "contessa_alembic_version"
+MIGRATION_TABLE = "contessa_alembic_version"
```

Comment:
could be `migration_table` and `version_table` be unified, if they mean the same thing?

Reply:
I would prefer MIGRATION_TABLE, because version_table is used inside Alembic migrations - in the Alembic context it is clear we mean version of migration. In context outside Alembic, in the Contessa we have package versions and migrations versions, so that version_table could be misleading.

Reply:
Ok, just thinking out loud.